package.json

```diff
@@ -49,7 +49,6 @@
     "caniuse-api": "^1.5.3",
     "chai": "^3.3.0",
     "coveralls": "^2.12.0",
-    "es6-shim": "^0.35.3",
```

Comment:
Oh, interesting. I guess if it's available in node 4, it's probably fine for us to remove it.